src/ops/ops.go

```diff
@@ -181,6 +181,15 @@ func (o *ops) SetBootOrder(device string) error {
 		return nil
 	}
 
+	o.log.Infof("SetBootOrder, runtime.GOARCH: %s, device: %s", runtime.GOARCH, device)
+	if runtime.GOARCH == "ppc64le" {
```

Comment:
What about s390x? Do we need to handle it also? Do we have another task for it?

Reply:
Let me check with Z  team about it.

Reply:
Z system does not has such cmd to set boot order inside RHCOS.